ONSdigital/dp-api-clients-go
# Investigate re-usage of Clienters among different API clients.

### What

We want to have the capability to re-use a dp-net/http Clienter among different API clients. In this PR, there is the suggested implementation in Image API client, that all clients should follow.

### How to review

- Make sure that the new constructor function in ImageAPI and Health clients allows a Clienter to be reused
- Check if the changes in the README are clear
- Make sure unit tests pass

### Who can review

Anyone
